packages/ui/src/app/pages/Proposals/ProposalPreview.tsx

```diff
@@ -52,6 +56,9 @@ export const ProposalPreview = () => {
   const votingRounds = useVotingRounds(proposal?.votes, proposal?.proposalStatusUpdates)
   const [currentVotingRound, setVotingRound] = useState(0)
 
+  const { opening } = useOpening(urlParamToOpeningId(id))//
```

Comment:
Also this is wrong the url parameter is a proposal id not a opening id. As a result all proposal pages will show a 404. E.g: https://dao-git-fork-chrlschwb-fix-3903-gracingperiod-joystream.vercel.app/#/proposals/preview/606